Azure.ResourceManager.Core/Subscription.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.Core
+{
+    public class Subscription : SubscriptionOperations
```

Comment:
Can you add a scenario test that spits out some other property of the subscription data other than Id like DisplayName just so we know end to end its working